src/lib/y2firewall/widgets/proposal.rb

```diff
@@ -200,5 +200,30 @@ def help
         )
       end
     end
+
+    # Widget to set SELinux mode
+    class SelinuxMode < CWM::ComboBox
+      def initialize(settings)
+        textdomain "firewall"
+
+        @settings = settings
+      end
+
+      def label
+        _("SELinux Mode")
```

Comment:
As said, maybe in the context of the proposal  "SELinux" here is a bit redundant.

Reply:
yeah, we also discuss it with @kobliha my argument is that SELinux Mode is easily googable, on other hand selinux is already in frame. So looks like you see it also duplicite, so lets remove it.

Reply:
> SELinux Mode is easily googable

Should we add some useful help to the dialog?